projects/views.py

```diff
@@ -68,6 +68,34 @@ def project_member_or_admin_or_superuser(user, project, project_user):
     return False
 
 
+def can_manage_project_membership(user, project):
```

Comment:
These three functions mean 3 calls to Keycloak will happen whenever this page loads. I think we should instead have a general function to get the scopes/roles for the user/group combination, and then set the flags in the context like this:

```
"can_manage_project": "manager" in scopes,
"can_manage_project_membership": "manager" in scopes,
```